components/display/waveshare_epaper.rst

```diff
@@ -49,6 +49,11 @@ configuration.
 .. code-block:: yaml
 
     # Example configuration entry
+    font:
+      - file: 'some_font_file.ttf' # place the "some_font_file.ttf" in /config/esphome
```

Comment:
Suggestion to use an actual font name, and removing the comment (as the shown folder is very specific to a certain installation method and thus not universal).

```suggestion
      - file: 'fonts/Comic Sans MS.ttf'
```